### PR TITLE
Dashboard global readme oliver

### LINKED ISSUE #237: Finish Dashboard README
As a developer, I want clear documentation that outlines the installation process, the structure of the pipeline, and how to run the app, so new contributors can easily get started and understand the functionality of the app.

Update the readme to include details on:
- How everything works
- Wireframe diagram

### PR DESCRIPTION
**Pull request by olliejth**
Closes #237 

Added descriptions of different dashboard files.

### CODE REVIEW

File: dashboard/README.md [hunk 1]
```diff
@@ -75,19 +75,31 @@ pytest --cov
 
 ## How it works
 #### `aurora_map.py`
+- Uses the `geopandas`, `maltplotlib` and `psycopg` to generate different map visualisations that show:
```
Comment (MurasatoNaoya) [inline dashboard/README.md:78]: Clear listings of the dependencies and how they are used in each script.

File: dashboard/README.md [hunk 2]
```diff
@@ -75,19 +75,31 @@ pytest --cov
 
 ## How it works
```
Comment (MurasatoNaoya) [inline dashboard/README.md:76]: Good use of markdown for consistent formatting and separating sections.

**Review by MurasatoNaoya — APPROVED** [review]
LGTM.

File: dashboard/README.md [hunk 3]
```diff
@@ -75,19 +75,31 @@ pytest --cov
...
 #### `aurora_map.py`
+- Uses the `geopandas`, `maltplotlib` and `psycopg` to generate different map visualisations that show:
+    - Aurora activity
+    - General visibility
+    - Celestial body visibilities
 #### `azimuth_plot.py`
-#### `hold.py`
```
Comment (CalebMathers) [inline dashboard/README.md:79]: Hold that, hold

File: dashboard/README.md [hunk 4]
```diff
@@ -75,19 +75,31 @@ pytest --cov
...
+- The master file for running the StarWatch Streamlit Dashboard.
+- Pulls from separate page files contained within the [pages](pages/) directory to assemble the final dashboard.
+- Contains functionality for both reading from and writing to the central RDS database using the `psycopg2` module.
+- Handles user input and subscriber sign-up from the running dashboard page.
+- Uses pre-defined functions from: `load_dashboard_data.py`, `nasa_pipeline`, `aurora_map.py`, `azimuth_plot.py`.
 #### `load_dashboard.py`
+- Uses the `psycopg2` module to connect to the central RDS database and extract data.
 #### `nasa_pipeline.py`
+- Checks for todays NASA image of day and the current ISS coordinates.
+- This is done using both the [NASA API](https://api.nasa.gov/) and the [ISS API](http://open-notify.org/Open-Notify-API/ISS-Location-Now/).
```
Comment (CalebMathers) [inline dashboard/README.md:94]: Good links to have in the read me

File: dashboard/load_dashboard_data.py [hunk 5]
```diff
@@ -16,13 +18,13 @@ def connect_to_db():
...
         database=os.getenv('DB_NAME'),
-        port = os.getenv('PORT'),
+        port=os.getenv('PORT'),
         user=os.getenv('DB_USER'),
         password=os.getenv('DB_PASSWORD')
     )
 
 
-def load_from_starwatch_rds(conn:Psycopg2Connection, table_name:str) -> pd.DataFrame:
+def load_from_starwatch_rds(conn: Psycopg2Connection, table_name: str) -> pd.DataFrame:
```
Comment (CalebMathers) [inline dashboard/load_dashboard_data.py:27]: Looks nicer with spacing, good

**Review by CalebMathers — APPROVED** [review]

**Pull request merged**